src/FluentlyHttpClient/FluentHttpRequestBuilderExtensions.cs

```diff
@@ -60,6 +60,50 @@ public static class FluentHttpRequestBuilderExtensions
 		/// <returns>Returns request builder for chaining.</returns>
 		public static FluentHttpRequestBuilder AsPatch(this FluentHttpRequestBuilder builder) => builder.WithMethod(HttpMethodPatch);
 
+		/// <summary>
+		/// Set request as a graphql.
+		/// </summary>
+		/// <returns></returns>
+		public static FluentHttpRequestBuilder AsGql(this FluentHttpRequestBuilder builder, string query) => builder.AsPost().WithBody(new { query });
+
+		/// <summary>
+		/// Set request as graphql
+		/// </summary>
+		///<returns>Returns request builder for chaining.</returns>
+		public static FluentHttpRequestBuilder AsGql(this FluentHttpRequestBuilder builder, GqlQuery query) => builder.AsPost().WithBody(query);
+
+		/// <summary>
+		/// Creates a request using graphql.
+		/// </summary>
+		/// <returns>Returns request builder for chaining.</returns>
+		public static FluentHttpRequestBuilder CreateGqlRequest(
+			this IFluentHttpClient fluentHttpClient,
+			string query,
+			string endPoint = "api/graphql"
```

Comment:
i was thinking to actually remove the uri "shorthand" altogether since it should be simply configured globally instead of per request, and if you need to per request (for whatever reason) you can still use `.WithUri`. (same for all other methods)